developer-portal/for-developers/custom-integration/fast-sdks/ios/mobile-sdk.mdx

```diff
@@ -0,0 +1,219 @@
+---
+title: "Fast Mobile SDK Docs: iOS SDK"
+description: Getting Started with the Fast iOS SDK
+keywords: Fast Mobile SDK, Fast iOS SDK
+---
+
+import {
+  MobileIntroAndPrereqs,
+  MobileInvokeCheckout,
+  MobileProductOptionsWarning,
+  MobileSignout,
+} from "../../../../../components/MobileMDXMacros.tsx";
+
+<MobileIntroAndPrereqs
+  sdkFullName="Fast iOS SDK"
+  shortOS="iOS"
+  sdkModuleName="FastKit"
+  sdkLanguage="Swift"
+  sdkCompatibleLanguage="Objective-C"
+  versionsSupportedOS="iOS 12.1 and above"
+  supportedDevices="both iPhone and iPad"
+/>
+
+# Set Up the SDK
+
+Getting started with the iOS SDK requires four steps:
+
+1. [Install the SDK](#install-the-sdk)
+2. [Configure your app](#configure-your-app)
+3. [Use Fast Checkout](#use-fast-checkout)
+4. [Handle Fast Events](#handle-fast-events)
+
+## Install the SDK
+
+You have several options on how to install the Fast SDK, described below.
+
+### With CocoaPods
+
+1. If you haven't already, [install the latest version of CocoaPods](https://guides.cocoapods.org/using/getting-started.html).
+2. In the terminal, run the following command from your project root to create a _Podfile_:
+
+   ```bash
+   pod init
+   ```
+
+3. Add this line to your Podfile:
+
+   ```bash
+   pod 'FastKit'
+   ```
+
+4. Add dependencies and generate an .xcworkspace:
+
+   ```bash
+   pod install
+   ```
+
+### As .xcframework
+
+A package manager like Cocoapods is recommended to ensure easy updates, but Fast can also be added to your app directly.
+
+:::warning DOWNLOAD LINK MISSING
+Let me know how to link
+:::
+
+1. Download the latest version of **FastKit.xcframework**
+2. Drag and drop the **FastKit.xcframework** file into the project navigator in XCode, and ensure all relevant targets are selected.
+3. Navigate to your project's settings. In the _General_ tab, add **FastKit.xcframework** under the _Frameworks, Libraries, and Embedded Content_ section. Make sure “Embed & Sign” is selected.
+
+## Configure Your App
+
+<embed src="/reusables/for-developers/_platform_all_sign_up_as_a_seller_banner_sandbox_and_contact_support.md" />
+
+In your AppDelegate's _didFinishLaunchingWithOptions_(...) method, configure FastKit with your **Fast App Id**:
+
+```swift
+Fast.shared.configure(appId: "YOUR_FAST_APP_ID")
+```
+
+On pages where you want the **FastCheckoutButton** to appear, generate the button and add it to your view hierarchy.
+
+```swift
+// Create and add the Fast Checkout Button
+let button = Fast.shared.createFastCheckoutButton()
+view.addSubview(button)
+
+// Note: Don't forget to handle the tap action!
+button.addTarget(self, action: #selector(invokeFastCheckout), for: .touchUpInside)
```

Comment:
putting `invokeFastCheckout` here makes it sound like this is a function we provide when this is really supposed to be their code. It might be more clear to just say something like: `The button does not have an action installed by default. Instead you should add your own action that calls into one of Fast's checkout functions.` and we could link to the next section which describes those functions.

Reply:
@tsabend 
Is there a minimal/barebones implementation we could provide for `invokeFastCheckout()` definition ... I could put a big banner / admonition saying that it should be filled out more for production code (or just provide minimal definition + pseudocode in code block comments), but just so an external dev won't need to guess when they're first testing out the SDK?

Reply:
maybe something like:
```swift
// Create and add the Fast Checkout Button
let button = Fast.shared.createFastCheckoutButton()
view.addSubview(button)
button.addTarget(self, action: #selector(invokeFastCheckout), for: .touchUpInside)
...
@objc func invokeFastCheckout() {
  // Call into one of the overloads for the checkout function.
  Fast.shared.checkout(...)
}